### DISCUSSION TITLE
The zero from which an instrument file's Positions are referenced in the same, whether for fingering holes or the splitting edge, correct? (0=Top of bore)

### DISCUSSION THREAD

**Discussion by JeffBurris** [edwardkort/WWIDesigner#121] [NAF Study] [answered]
As always, thanks for the software and the support.
There is no doubt in my mind I heard the answer to this in the videos somewhere but I cannot pinpoint it again.

Are the finger hole positions measured from the same zero point as the splitting edge position -- namely the top or Northern wall of the bore?
I have zero backset so, for example, my TSH Length and Splitting Edge Position are equal.
My TSH length is .260", I have no backset, so my splitting edge position is also .260".

Are the finger holes measured from the same zero? For some reason I thought they were not.

One might assume the zero point for the finger holes would be the same as anything else labeled "position", but for some reason I have this lingering memory (albeit possibly a false one, erroneously mixing memories of different methods?) of the fingering holes being measured either from the center of the TSH or the Splitting edge... ?

But for example in the nodal interference video I though you said the acoustic zero for a spreadsheet parameter was either the splitting edge or the top of the bore but I seem to be going crazy because now I can't find that again, either.

Now, in regard to NAFlutomat, you or someone wrote in item 3 under Some Details on Flutopedia:

3. You can measure the bore length to anything you want: splitting edge, back of TSH, or top of backset. But do it consistently. The difference is subsumed in the k2, calculated using the block and TSH factors you determined operationally.

...but NAFlutomat is a bit of a different animal in that the finger hole positions are made from the foot of the flute, so I assumed there _consistently_ meant from flute to flute, since you only measure the bore length once per unit.

I almost want to err on the side of making my holes a smidge larger (since I'm always battling my smallest hole wanting to be smaller than Mr. Prairie's spreadsheet recommends for a 1.4 octave flute) in which case I'd move a tinch South i.e. measure finger hole positions from the Splitting Edge instead of the Northern wall of the bore, but when I fiddle with anything improperly at all it tends to make tuning in my cross fingerings well impossible (it's often difficult for one or two of the lower cross fingerings not to come out too sharp, even when I cut or undercut a hole I'm tuning on its North side only). This is where I really relyt on WID -- making a reasonably Chromatic NAF. I currently have 14 notes active with a weight of one or more in my tuning files.

Am I complicating things and confusing memories?
Is the zero point for the finger holes positions the same zero point the splitting edge position is referenced from?
By default the zero for all Positions in the Instrument file view is the Northernmost wall of the bore?

Thanks!
Jeff

**Comment by edwardkort** [comment, marked as answer]
Hi Jeff,

Yes, the specification of position zero is covered in gory detail in the video: [4. WIDesigner Instrument Representation - NAF](http://youtu.be/7sdnDpqy278).

Short answer, the zero is the same for all the longitudinal dimensions: splitting edge, bore points, and holes. You can make that zero point the top of the bore, the top of the TSH, or the splitting edge - just by changing the value for the splitting edge position.


I interpret Mike Prairie's nodal interference ratio as from the splitting edge internally in WIDesigner. However, I'm not that fussy when entering values into the spreadsheet; I just add a couple of percent to the ratio it calculates.

As for tweaking hole sizes and positions, be a bit careful here. The tuning of the cross-fingerings is critically dependent upon the position of the holes; let the hole sizes be what they may during the tuning process. 

I consider the hole sizes that WIDesigner determines to still be a work in progress. A year ago I spent a great deal of time, in collaboration with several flute makers, in collecting real flute data and updating the tonehole model internal to WIDesigner to yield a more accurate hole size determination. Unfortunately, there still remained differences between predicted and measure hole sizes among the different flute makers. Fortunately, these differences are relatively small. It is still on my to-do list to investigate a more sophisticated tonehole model that, through empirical input, will make these differences disappear.

Hope I've answered your questions.

Cheers,
Edward Kort
Wood Wind

**Comment by JeffBurris** [comment]
Thanks so much, Mr. Kort!
I'm not sure why I got it in my head the zero was anything else for the fingering holes than for other longitudinal dimensions.
Just mixing memories I suppose.
Thanks on both counts -- yes, I thought I'd seen that somewhere, as well, that the nodal interference ratio (and perhaps a measurement involved in its calculation) was/were from the splitting edge in Mr. Prairie's spreadsheet, because it was truer acoustically speaking than using the top of the bore or top of the TSH. I am not nearly versed enough in acoustics to know exactly why. I just know the initial whistle sound is caused by the splitting edge but that the standing wave it creates in the bore presumably spreads out against the top of the bore, which gets complicated in my mind if one has any backset since the TSH would now be a hole in a rather strange place. 
I will enjoy learning more about acoustics and how they apply to woodwinds just as I have been enjoying playing with RF waves and antennas. 

I would not have guessed WID was not 100% accurate in hole size. Any difference in my real life results I'd chalked up to minor variations in my bore, wood grain, or other hidden variables in individual flutes that would one day have someone claiming the results were off even if WID's results had become technically perfect.

The fipple factor has been a pure blessing because as I am playing with attaching a separately-made nest to a flute (complete with flue, SAC hole, and TSH plus block retainer ridges). Inserting this part sometimes alters the dimensions of the bore just a little. I could approximate this on paper and try to change the diameter of the bore there as added positions, but things like this (esp. having to do with the area around the splitting edge) seem to get absorbed just fine into the Fipple Factor. 

I even blamed WID (silently, luckily, since the blame was misplaced) for not accurately lending itself to the in-tune cross fingerings it predicted, but it turns out I simply needed to stop drifting centers of finger hole  position as I manually enlarged holes. I chnaged how I go about that and things are pretty fantastic. Either the first or second lowest cross fingering likes to give me a scare but it's just the nature of the beast and I see there's a note or two infamous for this.

Thanks again for all you've done and do for this and related projects
Jeff Burris
at Otter Lake
Odessa, FL

**Comment by edwardkort** [comment]
I'm glad what I explained made sense, Jeff.

Any backset above the TSH acts quite a bit differently than just a bore extension. I modeled it as a simple Helmholtz resonator (the basic model for an ocarina); it has frequency component different than just a bore extension. So adding/subtracting headspace can be useful in tuning second-octave notes.

Yes, the Fipple Factor really serves as a catch-all for tonal contributions that are virtually impossible to model and measure separately.

Drop-in sound mechanisms are an interesting construction. I just started into 3D printing, and my first flute attempt had a wooden drop-in. But my typical flute is made of many separate components (as opposed to crafting all but the bird in the flute body), so for me a drop-in just adds a separate nest.

Again, I'm pleased that I can help.

Cheers,
Edward Kort
Wood Wind
